docs/components/movement-sensor/gps/gps-nmea.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Configure an NMEA-based GPS"
+linkTitle: "gps-nmea"
+weight: 10
+type: "docs"
+description: "Configure an NMEA-based GPS."
+# SMEs: Rand
+---
+
+A global positioning system (GPS) receives signals from satellites in the earth’s orbit to determine where it is and how fast it is going.
+All supported GPS models provide date for the `Position`, `CompassHeading` and `LinearVelocity` methods.
+Fix and Correction data are available by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](../../../sensor/).

```

Comment:
I think this is on mulltiple pages.

Reply:
```suggestion
You can obtain fix and correction data by using the sensor `GetReadings` method, which is available because GPSes wrap the [sensor component](../../../sensor/).
```